aws/cloudformation-templates/services/service/service.yaml

```diff
@@ -113,7 +113,15 @@ Resources:
               Service: ecs-tasks.amazonaws.com
       ManagedPolicyArns:
         - arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy
-
+      Policies:
+        - PolicyName: ReadRetailDemoStoreSSMParams
```

Comment:
Since this is only needed for accessing secure strings, consider renaming PolicyName so it's more clear. Otherwise it's potentially confusing why the TaskRole doesn't already cover this. Alternatively, add a CFN comment to make it clear why this is needed on the execution role. Thinking about maintenance and potential for something thinking it's not needed and removing it in the future.